rpcs3/Emu/Cell/lv2/sys_gpio.cpp

```diff
@@ -0,0 +1,36 @@
+#include "stdafx.h"
+#include "Emu/Memory/Memory.h"
+#include "Emu/System.h"
+#include "Emu/IdManager.h"
+
+#include "Emu/Cell/ErrorCodes.h"
+#include "sys_gpio.h"
+
+namespace vm { using namespace ps3; }
+
+logs::channel sys_gpio("sys_gpio");
+
+error_code sys_gpio_get(u64 device_id ,vm::ptr<u64> value)
+{
+   sys_gpio.trace("gpio get (device_id = 0x%x ,value = *0x%x)",device_id ,value);
```

Comment:
You use 3 spaces as indentation, but rpcs3 uses tabs of lengh 4. Check [editorconfig](https://github.com/RPCS3/rpcs3/blob/master/.editorconfig).

Also, I recommend you to install [EditorConfig extension ](https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig)for Visual Studio to let the IDE do this for you.